src/test/java/org/opentripplanner/street/model/edge/StreetEdgeTest.java

```diff
@@ -121,6 +128,115 @@ public void testTraverseAsCar() {
     assertEquals(expectedWeight, s1.getWeight(), 0.0);
   }
 
+  @Test
```

Comment:
Since this test is already quite crowded and you don't depend on the setup method, I would like to see this moved into another Java file. There are already a few other StreetEdge*Test files which test the various aspects.

Reply:
I moved the other new tests to a new class as well. Not sure if it was ok or not.